spark/dl/src/main/scala/com/intel/analytics/bigdl/models/utils/ModelBroadcast.scala

```diff
@@ -115,6 +122,7 @@ private[bigdl] class ModelBroadcastImp[T: ClassTag](applyProtoBuffer: Boolean =
       putWeightBias(SerializationUtils.clone(weightsBias), model)
       initGradWeightBias(weightsBias, model)
     }
+    this.setNodeAndCore()
```

Comment:
remove `this` and `()` ? to keep common style

Reply:
ok, changed